saleor/graphql/core/utils/__init__.py

```diff
@@ -37,11 +39,31 @@ def validate_image_file(file, field_name):
     """Validate if the file is an image."""
     if not file:
         raise ValidationError(
-            {field_name: ValidationError("File is required", code="required")}
+            {field_name: ValidationError("File is required.", code="required")}
         )
     if not file.content_type.startswith("image/"):
         raise ValidationError(
-            {field_name: ValidationError("Invalid file type", code="invalid")}
+            {field_name: ValidationError("Invalid file type.", code="invalid")}
+        )
+    _validate_image_format(file, field_name)
+
+
+def _validate_image_format(file, field_name):
+    """Validate image file format."""
+    Image.init()
+    allowed_extensions = [ext.lower() for ext in Image.EXTENSION]
+    _file_name, format = os.path.splitext(file._name)
+    if not format:
+        raise ValidationError(
+            {field_name: ValidationError("Lack of file extension.", code="invalid")}
```

Comment:
IMO The code used in ValidationError, should be taken from proper enum. If the validate_image is used in many different mutations you can pass an expected ErrorEnum class as an argument to validate_image function.